en/sql/schema/index_stmt.rst

```diff
@@ -227,6 +240,84 @@ Online unique index while other transactions inserts violates uniqueness
 |                                                                   |                                                                                   |
 +-------------------------------------------------------------------+-----------------------------------------------------------------------------------+
 
+.. _deduplicate_overview:
+
+DEDUPLICATE 
+-----------
+
+If you use the **DEDUPLICATE** option, you can improve performance degradation that can occur when modifying index data due to skewed data that is biased toward a specific key value. The value of this option can be adjusted to mitigate the overflow page's linked list for a specific key value from being made too long, thereby improving insert/delete/update and vacuum performance. However, since an index column hidden by the system is added, the number of terminal nodes in the index and the height of the tree may increase, increasing the size of the index and affecting search performance. In particular, if the index data is uniformly distributed with respect to key values, care must be taken because only the size of the index may increase without improving performance.
```

Comment:
```suggestion
If you use the **DEDUPLICATE** option, You can improve performance of modifying skewed index. The value of this option can be adjusted to mitigate the overflow page's linked list for a specific key value from being made too long, thereby improving insert/delete/update and vacuum performance. However, since an index column hidden by the system is added, the number of leaf nodes in the index and the height of the b-tree may increase, increasing the size of the index and affecting search performance. In particular, if the index data is uniformly distributed with respect to key values, you are careful to use this option because only the size of the index may increase without improving performance.
```

Reply:
Thank you,